### PR TITLE
Marius/sort date

### PR DESCRIPTION
**Pull request by EnvyPhD**
**Fix**
- Styles now override Ant-defaults.
- index.css -> index.scss.
- DOM now has correct nesting of elements.

**Changes**
- Changed every color-field to us SCSS variables.
- Changed size of Card to diversify from table, and clear whitespace.
- Introduced accent color and changed some elements to use it.
- Increased visibility of breadcrumbs.
- Updated colorscheme
- Removed double search-text (we should adhere to the field moving up above the box when clicked/typed)

**Add**
- Implemented SCSS variables for colors across all SCSS files.
- Sorting of table on date now works.
- Icons to Card to alleviate whitespace and make it less boring.

### CODE REVIEW

File: src/dataexplorer/dataexplorer-application/src/components/Dashboard/Dashboard.style.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import '../../variables';
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Dashboard/Dashboard.style.scss:1]: Possible to remove '../' here?

```suggestion
@import 'variables';
```

File: src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/Breadcrumbs.style.scss [hunk 2]
```diff
@@ -1,11 +1,20 @@
+@import '../../../variables';
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/Breadcrumbs.style.scss:1]: Same here?

```suggestion
@import 'variables';
```

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss [hunk 3]
```diff
@@ -1,13 +1,15 @@
+@import '../../../../../variables';
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss:1]:
```suggestion
@import 'variables';
```

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx [hunk 4]
```diff
@@ -60,68 +69,100 @@ const PatientView = (props: {
...
+                                    <h4>
+                                        <IdcardOutlined className="field-icon" />
+                                        Personnummer:
+                                    </h4>
 
+                                    <p>{props.patient.identifier[0].value}</p>
+                                </div>
+                                <div className="item-container">
+                                    <h4>
+                                        <ManOutlined className="field-icon" />
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:92]: Should this not only be the man's gender icon but depend on gender or be gender-neutral?

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx [hunk 5]
```diff
@@ -60,68 +69,100 @@ const PatientView = (props: {
...
+                                        {props.patient.birthDate !==
+                                        undefined ? (
+                                            calcAge()
+                                        ) : (
+                                            <div>Ikke oppgitt</div>
+                                        )}
+                                    </p>
+                                </div>
+                            </div>
+                            {console.log(props.patient)}
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:122]: Ohohh 😮

File: src/dataexplorer/dataexplorer-application/src/components/Dashboard/Dashboard.style.scss (hunk 1 above)
Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Dashboard/Dashboard.style.scss:1]: I can take a look

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx (hunk 5 above)
Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:122]: what in the world :|||||| will remove

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx (hunk 4 above)
Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:92]: I was hoping to find an icon for multigender, but ant design doesnt have it. I could either make it conditional on m/f/x, or i could find an icon that suits all posibilities from another source. What do you think? Personally i think the latter would fit more.

Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:92]: For example one of these: https://www.iconfont.cn/search/index?searchType=icon&q=gender

Comment (froydis98) [reply src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:92]: Agree, If you don't find an icon in exact same style I have previously gotten help from Idean to design the icon!

Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:92]: For now, i just made a small conditional for male/female. The API doesnt seem to accept other values to the gender field, so this is sufficient for now. Will expand later if needed.

**Comment by EnvyPhD** [comment]
Conditional render now added. Other fixes also implemented.

**Pull request merged**